### ISSUE TITLE
Please add a Bookshare link to the Precalc and Intermediate Algebra details pages

### ISSUE THREAD

**Issue by bonnebbrown** [openstax/bit#106]
### Description

Bookshare is available for OS titles: Precalc and Intermediate Algebra (as mentioned in the Accessibility slack channel) but they are not linked to Bookshare via their details pages. 
Not sure if you all need the links, but...
Precalc details: https://openstax.org/details/books/precalculus
Bookshare: https: https://www.bookshare.org/browse/book/2292178
Intermediate Algebra Details: https://openstax.org/details/books/intermediate-algebra
Bookshare: https://www.bookshare.org/browse/book/2292109

****FYI: I found Psychology on Bookshare too, but you need a membership to access it (which may cost $) and I’m not sure we’d want to link to that: https://www.bookshare.org/browse/book/1824739
Also, I don’t know about the Chemistry text (as mentioned in the accessibility channel) bc the title says "Chemistry- Adaptive Course With Homework": https://www.bookshare.org/browse/book/2292177
----------------
**Steps to Reproduce the bug or Details about Change:** _What, Who, Where and How?_

- **User Type:**
` [ ] ` Admin  ` [ ] ` Content/QA
` [ ] ` Student  ` [ ] ` Teacher
` [ ] ` N/A
- **Username:** 
- **URL:**  
- **Course number:**  
- **Course type:**  

- **Steps**

----------------
**Environment:**

- **Instance:**
` [ ] ` production ` [ ] ` qa ` [ ] ` staging
` [ ] ` dev ` [ ] ` content ` [ ] ` demo
- **OS:**
` [ ] ` Mac OSX ` [ ] ` Windows
` [ ] ` Android ` [ ] ` iOS
- **Browser:**
` [ ] ` Chrome ` [ ] ` Firefox ` [ ] ` Safari
` [ ] ` Opera ` [ ] ` IE ` [ ] ` Edge
- **Browser Version:**  

----------------
**Possible impact on a User:**
"Additionally, some of our titles are available through Bookshare in BRF and/or DAISY formats to extend accessibility." from OS Accessibility Statement: https://openstax.org/accessibility-statement
----------------
**Images or Screenshots** (_attach below_)

**Comment by gregfitch** [comment]
If a change is needed, please make sure it is made to all environments.

**Comment by allisonhowell** [comment]
@laurenloh Could you please add them on the dev, cms-qa, and staging sites as well? See Greg's comments above.

**Comment by allisonhowell** [comment]
@laurenloh did you add to staging? You can close when that is done.
